packages/editor/src/client/panel/peerEditPanel.tsx

```diff
@@ -220,7 +220,13 @@ export function PeerEditPanel({id, hostURL, onClose, onSave}: PeerEditPanelProps
                 {profile?.name}
               </DescriptionListItem>
               <DescriptionListItem label={t('peerList.panels.themeColor')}>
-                {profile?.themeColor}
+                <div
```

Comment:
I recommend to still display the hex code of the theme color.

Reply:
Ok I'll change it back, can you tell me why it's better with the hex code ?